extensions/blocks/gif/editor.scss

```diff
@@ -79,6 +81,6 @@
 		max-width: 200px;
 	}
 	svg path {
-		fill: #ccc;
+		fill: $dark-gray-150;
```

Comment:
Before:

<img width="309" alt="Screenshot 2019-04-26 at 16 48 30" src="https://user-images.githubusercontent.com/87168/56816358-0d36a900-684c-11e9-8079-fb37ea06707b.png">

After:

<img width="308" alt="Screenshot 2019-04-26 at 16 48 21" src="https://user-images.githubusercontent.com/87168/56816362-10ca3000-684c-11e9-8097-758effbdabc3.png">

The new tone is _"lightest gray that can be used for AA non-text contrast."_

Reply:
FYI @jeffersonrabb